### ISSUE TITLE
The guide of Install the AntiSmash-5.1.2 

### ISSUE THREAD

**Issue by Oldmagua** [antismash/antismash#290]
the command i used as follow:

```
conda create -n  antismash python=3.7
conda activate antismash
conda install hmmer2 hmmer diamond fasttree prodigal blast muscle glimmerhmm
```
# the python version could be problem for installing antismash , python 3.8.3 seems not satisfied with antismash5 ,  so here we set the python version to 3.7.8, and everything goes well~
`pip install antismash-5-1-2 `# i had already download antismash 5.1.2 ,if not , you should download the package first!
`download-antismash-databases` #here maybe you have trouble with the download database with internet time out, you can download the database to local computer first ,then upload to the server .

the databese-file place :
```
cd /miniconda3/envs/antismash/lib/python3.7/site-packages/antismash/databases
mkdir -p pfam/27.0
cp Pfam27.0/Pfam-A.hmm.gz pfam/27.0
mkdir -p pfam/31.0
cp Pfam31.0/Pfam-A.hmm.gz pfam/31.0
mkdir -p resfam
cp resfams/Resfams.hmm.gz resfam
cp clusterblast_20190415.tar.xz ./
```

run `download-antismash-databases` again~

then you will successfully install the antismash-5.1.2 and the database！

**Comment by drelo** [comment]
This recipe worked some days ago for me. Now I am trying again in a new environment and I got

```
ERROR: Could not find a version that satisfies the requirement antismash-5-1-2 (from versions: none)
ERROR: No matching distribution found for antismash-5-1-2
```
following those instructions. As a workaround I tried 'conda install -c bioconda antismash' and It throws some error too. Are there recent changes affecting this? I am trying this since I found the "Bio.Alphabet" error following other procedures. 
I tried  `pip install yolk3k`  `yolk -V antismash` and I couldn't find any version through this.
